THIRD-PARTY-NOTICES.TXT

```diff
@@ -322,3 +322,185 @@ SHALL THE COPYRIGHT HOLDERS OR ANYONE DISTRIBUTING THE SOFTWARE BE LIABLE
 FOR ANY DAMAGES OR OTHER LIABILITY, WHETHER IN CONTRACT, TORT OR OTHERWISE,
 ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 DEALINGS IN THE SOFTWARE.
+
+-----------------------------------------------------------------------------------------------------------------------------------------
+Source in src/Tools/Common/WindowsProcessExtension/WindowsProcessExtension.cs was copied from 
```

Comment:
Now that the code has been refactored this isn't the correct description. In this file you could just say "License notice for code based on https://github.com/projectkudu". Then in the multiple source files where that copied code resides you can comment on where exactly it came from.

Reply:
Is there a way to delete or remove a pull request?

Reply:
You can close it without merging in the code, but I am not aware of any way to delete it. Click the 'Close pull request' button at the bottom of the page.